cfgov/jinja2/v1/sublanding-page-2/index.html

```diff
@@ -10,16 +10,29 @@
 
 {#
     TODO: Add the following molecules
+        - Hero
         - Post feed
         - Grey well
         - Sidebar with related content and mailing list
 #}
 
+{# TODO: Remove lipsumText when and if it is no longer needed. #}
+{% set lipsumText = lipsum(1, false, 10, 25) | safe %}
+
 {% block content_main %}
     <div class="block
                 block__flush-top">
         {# TODO: Add main molecules and organisms. #}
-        {{ hero.render() }}
+
+        {{ main_contact_info.render( {
+            'heading': 'Whistleblower tips',
+            'body': lipsumText
+        }, {
+            'emails':  ['test@example.com'],
+            'phones':  [{'number': '1234567899',
+                         'vanity': '123456AAAA',
+                         'tty': '1234567899'}]
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/8185f1710e4fd57e151ac2858919f7d3e3874f22